<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QWidget" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>358</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>528</width>
    <height>358</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>528</width>
    <height>358</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelImageHolder">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>331</width>
     <height>321</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>331</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Тут должно быть
изображение-инструкция.</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>171</width>
     <height>156</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <item>
     <widget class="QComboBox" name="cmbboxChooseCOM">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string/>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToMinimumContentsLength</enum>
      </property>
      <property name="placeholderText">
       <string>Выберите COM-порт</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_12">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnUpdateCOMPorts">
      <property name="text">
       <string>Обновить список COM-портов</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_13">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnChooseFile">
      <property name="text">
       <string>Выберите файл</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnStartTest">
      <property name="text">
       <string>Начать тестирование</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="labelInfo">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>169</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Информация о COM-порте:
Скорость работы:
Количество бит:
Контроль четности:
Стоп бит:
Вид управления потоком:
Номер теста: 0 из 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFilename">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>149</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя файла:
</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCOMportName">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Название COM-порта:
</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
